pom.xml

```diff
@@ -129,6 +129,28 @@
                 </pluginManagement>
             </build>
         </profile>
+        <profile>
+            <id>sonar-cloud</id>
+            <properties>
+                <!-- SonarQube rule exclusions for: Integration tests -->
+                <sonar.issue.ignore.multicriteria>e1,e2,e3,e4,e5</sonar.issue.ignore.multicriteria>
+                <!-- Disable: Public types, methods and fields (API) should be documented with Javadoc -->
+                <sonar.issue.ignore.multicriteria.e1.ruleKey>java:S1176</sonar.issue.ignore.multicriteria.e1.ruleKey>
+                <sonar.issue.ignore.multicriteria.e1.resourceKey>**/*-integration-tests/**/*.java</sonar.issue.ignore.multicriteria.e1.resourceKey>
+                <!-- Disable: Constructors should only call non-overridable methods -->
```

Comment:
Reported as code smell with high severity. This is a problem in general with Flow, where you often call `Component.add` in the constructor. Unlikely to result in an actual issue in the simple test setup classes that we have.